index.js

```diff
@@ -18,9 +20,8 @@ mongoose.connection.on('error', err => {
   console.log('Database error ' + err);
 });
 
-const app = express();
-
-// Express built-in middleware that does job of body-parser
+// HTTP logger & Express built-in middleware that does job of body-parser.
+app.use(volleyball);
```

Comment:
qq...What is `volleyball` used for?

Reply:
I think it's for logging http request and responses. It's a Middleware